packages/suite/src/storage/index.ts

```diff
@@ -89,7 +105,7 @@ export type SuiteStorageUpdateMessage = StorageUpdateMessage<SuiteDBSchema>;
  */
 const onUpgrade: OnUpgradeFunc<SuiteDBSchema> = async (db, oldVersion, newVersion, transaction) => {
     let shouldInitDB = oldVersion === 0;
-    if (oldVersion > 0 && oldVersion < 13) {
+    if (oldVersion > 0 && oldVersion < 16) {
```

Comment:
why deleting older versions? migration is not possible?

Reply:
Yes, the migration is implemented.

![image](https://user-images.githubusercontent.com/3112191/94924073-2ae65400-04bd-11eb-829d-4220f936b79c.png)